RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -472,6 +474,28 @@ void EcalUncalibRecHitWorkerMultiFit::run(const edm::Event& evt,
         }
         uncalibRecHit.setJitter(timerh);
         uncalibRecHit.setJitterError(0.);  // not computed with weights
+     
+     } else if (timealgo_ == kansasMethodCC) {
+
+        float startTime = -25;
+        float stopTime = 25;
+        EcalUncalibRecHitTimingCCAlgo computeCC(startTime, stopTime);
+
+        uncalibRecHit.setJitterError( 0. );                  	
+        float timeStep=1;
+        float tempt = 0;
+
+        std::vector<double> amplitudes;
+        for(unsigned int ibx=0; ibx<activeBX.size(); ++ibx) amplitudes.push_back(uncalibRecHit.outOfTimeAmplitude(ibx));
+        // seedTime = 0;
+
+        tempt = computeCC.computeTimeCC( *itdg, amplitudes, aped, aGain, fullpulse, uncalibRecHit);
+
+        uncalibRecHit.setJitter( -tempt );
+        if (tempt > stopTime-timeStep || tempt<startTime+timeStep)
+          uncalibRecHit.setJitterError( -timeStep/25 ); 
```

Comment:
Use named constant instead of 25.
If this 25 is the sampling period you can use the constant in EcalConstants: https://github.com/cms-sw/cmssw/blob/master/DataFormats/EcalDigi/interface/EcalConstants.h#L25